### DISCUSSION TITLE
Discussion: Create plan for short-form video content on Tiktok

### DISCUSSION THREAD

**Discussion by eidolonnight** [WordPress/Marketing-Team#374] [Ideas]
Early in 2022 we were able to claim our [Tiktok account (WordPress](https://github.com/WordPress/Marketing-Team/issues/7)), but have not yet published anything to the platform. Production of short-form video presents a number of unique challenges for our team.

Some questions for us to consider:
- What content and campaigns would be best suited to Tiktok?
- How can we sustainably produce quality content on a weekly cadence?
- Can and should this content match the design aesthetic of WordPress.org and our other channels?
- Where can videos be stored and handed off to a campaign manager with access to Sprout?

As part of a WCUS 2022 brainstorm, a few contributors (Matthew1551, craylor, jmchannell) helped [outline steps for production](https://docs.google.com/document/d/1PgCn3fl3aydTL8CPTbMoIorA2wa4iDzsmYivpgSCwYs/edit#). This could be published as a [handbook](https://make.wordpress.org/marketing/handbook/) guide for contributors, though we will still need to provide content guidance for content creators.

**Comment by sereedmedia** [comment]
> What content and campaigns would be best suited to Tiktok?

The first question that needs to be answered is who is the content for, so community? Users? Developers? A combination? 

Second question, what is the goal of the content being created? Some potential goals (of course depending on the first answer): 

- Get ppl to use wordpress (abstract, but would end with a "get wordpress" CTA
- Showcase cool things people are doing with WordPress (promote existing community on TT, spotlight photos from the openverse/photo directory, art from the Block Museum (not sure how active that is but it is there!), look at websites that are listed in the [Showcase](https://wordpress.org/showcase/).
- Drive contributions (takeovers on or crossposting from WordCamp contributor days and/or doing overviews of Make posts, especially highlighting requests for feedback, new features, etc.)
- Educate (do overviews of/ promote new Learn videos/tutorials ..  could engage folks who post videos or Learn folks to be a part of posting this content)

If these questions have been addressed elsewhere (e.g. for other social media platforms), then please link me, as I didn't find it through the Handbook link. 

> How can we sustainably produce quality content on a weekly cadence?
IMO this would best be achieved through getting content from other existing places in the WP community. So much content is being generated by the Make teams and the official community, it makes sense to me to amplify it.

Starting with content the marketing team is already doing, maybe even just putting out the newsletter on Tiktok (highlighting the info in the newsletter as backdrop as folks do with news articles) would be a great weekly baseline to start getting active. Also, that would be something that would be easy to keep consistent, and meet a lot of the goals I described above.

Could be fun to explore cross-promoting with WordCamp organizers as hosts, because there is usually a promotions person on the team who would be glad to host some content in order to promote the camp. Likewise, Meetup organizers. Would be nice to feature the faces of the existing community.

> Can and should this content match the design aesthetic of WordPress.org and our other channels?
Yes. Also, accessibility needs to be a top priority, in terms of captioning, etc.

> Where can videos be stored and handed off to a campaign manager with access to Sprout?
I've got nothing for this one! 😄

**Comment by murillol** [comment]
Agree with everything @sereedmedia said!

**Comment by nalininsbs** [comment]
Added for highlighting at 19 October 2022 meeting

**Comment by eidolonnight** [comment]
Sharing this cool content example from Squarespace: https://www.tiktok.com/t/ZTRuEUHda/

> Starting with content the marketing team is already doing, maybe even just putting out the newsletter on Tiktok (highlighting the info in the newsletter as backdrop as folks do with news articles) would be a great weekly baseline to start getting active. Also, that would be something that would be easy to keep consistent, and meet a lot of the goals I described above.

@sereedmedia I'd love to see examples of what you have in mind.

**Comment by eidolonnight** [comment]
> If these questions have been addressed elsewhere (e.g. for other social media platforms), then please link me, as I didn't find it through the Handbook link.

They have not! These are excellent questions and I'm curious what everyone thinks.

**Comment by sereedmedia** [comment]
> @sereedmedia I'd love to see examples of what you have in mind.

@eidolonnight 

I had to look for a while for something that wasn't too political to post lol, but here is an example of someone talking in front of a screen/news story, which is what I meant about putting out the newsletter on Tiktok:
https://www.tiktok.com/t/ZTRHaypgW/

So someone could do a video displaying the newsletter and talking through the highlights each week, with a link back to the archive. Promotes the newsletter and project and uses content already created.

**Comment by sereedmedia** [comment]
> Sharing this cool content example from Squarespace: https://www.tiktok.com/t/ZTRuEUHda/
> 
> >

I think that getting the website owner's view is definitely a good perspective. Highlighting people doing neat things with WordPress sites is cool too.

I do think, however, that Squarespace's target audience is much clearer. Their goal is to sell Squarespace accounts. The goal for WordPress.org is much less singular, because there is really nothing to "sell" other than the brand itself. 

That's why I think defining the audience and goals of the social accounts is so important. 

IMO, the real "product" that WordPress.org is able to/should "sell" is the community. WordCamps, Meetups, contribution. The idea that when you choose WordPress, you are becoming part of the community, whether you are a site owner, a dev, a marketer, whatever. 

Squarespace knows there is value in connection, so even in this video they are trying to be like "they've been with me the whole time" ... but Squarespace is and always will be just a website. WordPress is alive.

**Comment by eidolonnight** [comment]
> That's why I think defining the audience and goals of the social accounts is so important.

I'm not a tiktok expert, but it seems to me that there isn't a huge developer audience there, at least that I've found. The developers that are present seem to focus on "day in the life" and that kind of content. I assume this is due, at least in part, to the nature of the medium. Has this been everyone's experience or am I just following too many ducks and raccoons? 😄 

> I had to look for a while for something that wasn't too political to post lol, but here is an example of someone talking in front of a screen/news story, which is what I meant about putting out the newsletter on Tiktok:
https://www.tiktok.com/t/ZTRHaypgW/

This is a neat idea that'd be relatively "easy" IMO. We already have https://wordpress.org/news/category/month-in-wordpress/ so it's just a matter of someone doing videos. Seems like an easy way to dip our toes in.

**Comment by mumtahinafaguni** [comment]
I’ve a few ideas. We can follow Unicef tiktok channel. Moreover, here are some more ideas:
sharing hacks: such as, 

- How to change font color, add custom fonts, etc
- Showcase WordPress photos: create versatile reels using photos from WP photos
- Design hacks: promote patterns
- Encourage to capture moments in meetups, WordCamps, etc.
- We have sounds in Openverse. We can encourage to use those to create reels.

**Comment by nalininsbs** [comment]
This card is regularly highlighted in Marketing Team meetings in September, October and November 2022, and referenced in notes to raise awareness. Please continue to add your thoughts to the card.

@eidolonnight is there a timeline on this or is it early discussion phase? Thank you for starting this.

**Comment by sereedmedia** [comment]
I love the sharing hacks idea @mumtahinafaguni ! i think there is *so* much space for that! the Learn team would probably be eager to help with that as well! i think it's a great way to bring in new users.

i also love the sharing the Openverse content idea as well! there is so much "use this audio" posted, and that could be done very simply with a template. and that has the double win of promoting WP and Openverse. and of course there is https://wordpress.org/photos/, which would be rad to post with the audio! triple win? amazing.

also, all tiktok content can/should go on every other platform. so many ppl watch tiktok on reels.

**Comment by Afrin29** [comment]
I think if we just fix our target audience it will be a bit easier to create related content. Apart from this, here are some ideas I want to add. For example:

- We are doing weekly meetings & other stuff, so we can easily create one related event video each week and publish it so that the target audience can know how we are working and having fun. 

- We can assign one by one who is most active to share their experience overtime on the most relevant issues or ask questions on the marketing channel. Two things can happen with this: you can guide new joins a bit and let the target audience know how cooperative & helpful we are. 

- Random image sharing of meetings, works, fun moments, or events to know how we enjoy being a part of this community.

**Comment by nalininsbs** [comment]
Highlighted at a number of team meetings in October and November 2022.

**Comment by abhansnuk** [comment]
Could this be something to get further ideas at WCAsia contributor day @sereedmedia @bjmcsherry and @eidolonnight  ?

**Comment by sereedmedia** [comment]
I am wondering if we could formulate a plan out of the existing issue prior to WordCamp Asia and then have a basic call ready for new contributors to create video content during Contributor Day. Our main need to make that happen would be a repository for storing and tracking content that is created. (This content doesn't have to only be used on TikTok, but TikTok/Reels-formatted content would be primary goal.)

Do we have an existing TikTok account for tagging @bjmcsherry ?

**Comment by abhansnuk** [comment]
Technically you could have a board on this GitHub repo @sereedmedia for content that is created. I have added a temporary [test project board](https://github.com/orgs/WordPress/projects/69) to check this for you. But the storage location may depend on how the team that deal with the social media output store content / videos they are creating, and any anti-virus checks for video files necessary.

The Marketing-Team repo under WordPress should be the top level of under the WordPress organization for the Marketing Team. You can have multiple boards etc under it. This was to avoid multiple repos for Marketing sitting under the WordPress organization. (Note: at present, there is an issue with linking to the Marketing Teams board from the Marketing repo - as per previous conversations. This is to do with how the transfer to the organization was done, GitHubs beta boards and its options at the time. Since, training can link its newer boards directly under their repo. We currently do this as a manual link from the home of the repo)

For previous contributor days, we have created a special folder where content can be uploaded without people previously having joined marketing with google docs etc access set up. Once the items were completed or finished from the camp, the items were moved into the main Marketing drive.

**Comment by sereedmedia** [comment]
So a related content creation effort is happening via the @WordPress/learn-team and it would be great to align our efforts. They are discussing making "bite-sized" bits of training content for dissemination via WordPress social medias and other general marketing use: https://make.wordpress.org/training/2023/01/18/bite-sized-content-on-learn-wordpress/

Helping create topics and maybe even scripts for this endeavor would be great, and then hopefully those assets could get stored according to Learn's existing video infrastructure. We can also concurrently task out the marketing content needed for these snippets, such as captions, tags, hashtags, cta links, etc. 

And indeed, hopefully this plan could be ready to create and receive content on WCAsia Contributor Day. :)

**Comment by nomad-skateboarding-dev** [comment]
Really excited about this and all the progress being made! Thank you all for the efforts. 🙂🙌

I'd like to ask if it's possible to use the new Github projects and boards for any future changes? 

I'm currently unable to view the old projects format from GitHub mobile, and the links return a 404 error if followed to mobile browser.

**Comment by westnz** [comment]
Thank you for highlighting @sereedmedia. Here is the GitHub Issue we created with the first example added: https://github.com/WordPress/Learn/issues/1265

**Comment by nalininsbs** [comment]
> Really excited about this and all the progress being made! Thank you all for the efforts. 🙂🙌
> 
> I'd like to ask if it's possible to use the new Github projects and boards for any future changes?
> 
> I'm currently unable to view the old projects format from GitHub mobile, and the links return a 404 error if followed to mobile browser.

Hello @nomad-skateboarding-dev and thank you for the question. Can you share an example link you are having difficulties with. I have tried the project boards from two different mobile devices and can not replicate this error. The links open up for me. I emptied cache. I tried from an incognito window too. The project board, like many of the existing ones in the WordPress organization, used what GitHub had at that time. In future of any new boards, the new form of boards would be used. There are some reported accessibility and mobile issues with some examples of the new boards. I know some this is being looked into, and may resolve themselves.

**Comment by courtneyr-dev** [comment]
Should we consider using https://jetpack.com/support/jetpack-blocks/story-block/ on Learn.WordPress.org for the videos? We already use Jetpack on site. Ideally we could use the same content in vertical format (mobile) on:

- LearnWP
- TikTok
- Instagram reels
- FB reels

**Comment by meaganhanes** [comment]
It's extremely wise to focus on both horizontal and vertical formats for
the next stage of development IMO. Let's leverage Jetpack for now and see
how far it propels us.

**Comment by JosVelasco** [comment]
The sound of the first TikTok video was removed due to copyright restrictions.

I want to help produce TikTok videos. One idea would be to let the community members create their own and have rules like in https://wordpress.tv/ or https://wordpress.org/photos/ to moderate them.

**Comment by eidolonnight** [comment]
I originally created this as an issue, but it's not quite actionable yet. Converting to a discussion for now.
